packages-exp/auth-exp/src/core/strategies/custom_token.test.ts

```diff
@@ -67,31 +71,59 @@ describe('core/strategies/signInWithCustomToken', () => {
   });
   afterEach(mockFetch.tearDown);
 
-  it('should return a valid user credential', async () => {
-    const {
-      user,
-      operationType,
-      _tokenResponse
-    } = (await signInWithCustomToken(
-      auth,
-      'look-at-me-im-a-jwt'
-    )) as UserCredentialInternal;
-    expect(_tokenResponse).to.eql(idTokenResponse);
-    expect(user.uid).to.eq('local-id');
-    expect(user.displayName).to.eq('display-name');
-    expect(operationType).to.eq(OperationType.SIGN_IN);
+  describe('#signInWithCustomToken', () => {
```

Comment:
if I set a customTokenProvider and then call signInWithCustomToken, do I still have to pass a token? Is there not a version of signInWithCustomToken that will use my provider?

Reply:
Yeah, that's correct. There is not a version of `signInWithCustomToken` that doesn't take in a parameter. We could add one in the future, but that would need to go through another API review.

Reply:
ack